spirit/core/utils/markdown/block.py

```diff
@@ -47,6 +48,7 @@ class BlockGrammar(mistune.BlockGrammar):
         r'|youtu\.be/'
         r'|youtube\.com/embed/)'
         r'(?P<id>[a-zA-Z0-9_\-]{11})'
+        r'(&t=(?P<start_minutes>[0-9]+m)?(?P<start_seconds>[0-9]+s)?)?'
```

Comment:
* You would have to add those seconds manually. The YT sharing functionality will give you minutes when exceeding 59s
* See https://youtu.be/O1QQajfobPw?t=1h1m38s